connaisseur/validators/cosign/cosign_validator.py

```diff
@@ -255,33 +269,9 @@ def __invoke_cosign(self, image: str, key: str):
 
         return process.returncode, stdout.decode("utf-8"), stderr.decode("utf-8")
 
-    @staticmethod
-    def __get_pubkey_config(key: str):
-        """
-        Return a tuple of the used Cosign verification command (flag-value list), a
-        dict of potentially required environment variables and public key in binary
-        PEM format to be used as stdin to Cosign based on the format of the input
-        key (reference).
-
-        Raise InvalidFormatException if none of the supported patterns is matched.
-        """
-        try:
-            # key is ecdsa public key
-            pkey = load_key(key).to_pem()  # raises if invalid
-            return ["--key", "/dev/stdin"], {}, pkey
-        except ValueError:
-            pass
-
-        # key is KMS reference
-        if re.match(r"^\w{2,20}://[\w:/-]{3,255}$", key):
-            return ["--key", key], {}, b""
-
-        msg = "Public key (reference) '{input_str}' does not match expected patterns."
-        raise InvalidFormatException(message=msg, input_str=key)
-
     def __get_envs(self):
         """
-        Sets up environment variables used by cosign.
+        Set up environment variables used by cosign.
         """
         env = os.environ.copy()
         # Extend the OS env vars only for passing to the subprocess below
```

Comment:
Since I see it from here but Github doesn't allow me to comment there, ` __apply_policy` docstring descriptive